source/common/stream_info/stream_info_impl.h

```diff
@@ -435,6 +449,8 @@ struct StreamInfoImpl : public StreamInfo {
 
   std::shared_ptr<UpstreamInfo> upstream_info_;
   uint64_t bytes_received_{};
+  uint64_t bytes_retransmitted_{};
```

Comment:
bah.  I was going to ask for downstream-specific naming but looks like that ship has sailed =P